geomstats/geometry/discrete_curves.py

```diff
@@ -14,7 +15,7 @@
 R3 = Euclidean(dim=3)
 
 
-class DiscreteCurves(Manifold):
+class DiscreteCurves(OpenSet):
```

Comment:
Why didn't we have this as a ProductManifold?

Reply:
I don't think it would be efficient if `n_sampling_points` is large

Reply:
I don't think it would be efficient if `n_sampling_points` is large

Reply:
Good point!